common/src/main/java/ch/admin/bag/covidcertificate/common/qr/QrScanFragment.kt

```diff
@@ -184,6 +181,14 @@ abstract class QrScanFragment : Fragment() {
 				e.printStackTrace()
 			}
 		}, mainExecutor)
+		
```

Comment:
Maybe add a comment:
`// Set focus to the center of the viewfinder to help the auto focus`